### ISSUE TITLE
Observations on Thelio Mega

### ISSUE THREAD

**Issue by randomgeek78** [system76/thelio-io-windows#15]
I was able to compile the latest msi and install it as a service on the thelio Mega. When I restart, I see the following on my Event Viewer-

```
A device attached to the system is not functioning.
Custom {
    kind: Other,
    error: "A device attached to the system is not functioning.",
}
```

In the Services, I see `System76 Thelio Io` as running. The fan is quiet. 

But after some time, when the computer goes to sleep and is awoken, the service is not running anymore and I am not able to restart it. The fan at this point is very loud as understandable since the service is not running.

**Comment by randomgeek78** [comment]
After a reboot, the `System76 Thelio Io` gets restarted again and things are quiet.

**Comment by randomgeek78** [comment]
I was able to replicate the issue again - after the computer goes to sleep and is woken up, there is a loud fan noise. The `System76 Thelio Io` service shows as not running. When I try to start it, I get this message - 

![image](https://user-images.githubusercontent.com/66195352/188186204-b2394e88-690e-42da-a67c-a037efe1fc48.png)

In the event viewer, I see this recorded message-

```
Io command 'IoDUTYCPUF0E29' failed to write: A device attached to the system is not functioning. (os error 31)
Custom {
    kind: Other,
    error: "Io command 'IoDUTYCPUF0E29' failed to write: A device attached to the system is not functioning. (os error 31)",
}
```

**Comment by flintinatux** [comment]
@randomgeek78, were you able to find a solution to this?  I'm seeing similar behavior in Pop_OS now.

**Comment by randomgeek78** [comment]
@flintinatux No. I am still facing the problem. I just don't let the computer go to sleep anymore, only turn off the display when it is not needed. Otherwise, I shutdown the computer completely if I am not going to be using it for a while. I am hoping that this problem will get fixed and then go back to letting the computer sleep.
